python/health-check/src/health_check/__init__.py

```diff
@@ -2,4 +2,4 @@
 Health Check tool for Uyuni"
 """
 
-__version__ = "0.1"
+__version__ = "5.1.0"
```

Comment:
How will this be updated in the future?

Reply:
Ideally this should be upgraded automatically when running `tito tag`, in the same way the version in the spec is updated.

For that, we would need extend the current tagger used for `mgr-health-check` package via `rel-eng/custom/custom.py`, as it is done for other packages.

I'm having a look to this at the moment.